hitobito_skv.gemspec

```
@@ -0,0 +1,20 @@
+$LOAD_PATH.push File.expand_path('../lib', __FILE__)
+
+# Maintain your wagon's version:
+require 'hitobito_skv/version'
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |s|
+  # rubocop:disable SingleSpaceBeforeFirstArg
+  s.name        = 'hitobito_skv'
+  s.version     = HitobitoSkv::VERSION
+  s.authors     = ['Luca Brügger']
+  s.email       = ['lbruegger@puzzle.ch']
+  s.summary     = 'Skv organization specific features'
+  s.description = 'Skv organization specific features'
+
+  s.files = Dir['{app,config,db,lib}/**/*'] + ['Rakefile']
+  s.test_files = Dir['test/**/*']
```

Hier müssten wir wohl mal die Templates anpassen.

```suggestion
  s.test_files = Dir['(test,spec)/**/*']
```